napari/layers/base/base.py

```diff
@@ -710,22 +705,6 @@ def ndim(self):
         """int: Number of dimensions in the data."""
         return self._ndim
 
-    @property
```

Comment:
As commented in main thread, but be nice to deprecate, possibly by temporarily storing a copy of the layers privately on the layer!

Reply:
implemented